### ISSUE TITLE
Chat bubble bug in support version 25.1.0

### ISSUE THREAD

**Issue by tinashe96** [AppLozic/Applozic-Android-SDK#213]
When a message is sent, the chat bubble of the message does not appear in the conversation (on both sender and receiver side) unless I exit the conversation and re-enter the conversation.

**Comment by Sunilkumarr** [comment]
@tinOfBeans use the all support library versions 24.2.1 or 25.0.1

**Comment by adarshmishra** [comment]
@tinOfBeans , we have released new version of SDK with 25.1.0 support. 

```
compile 'com.applozic.communication.uiwidget:mobicomkitui:4.74'

```
 If you have imported our module code you can merge below changes:

https://github.com/AppLozic/Applozic-Android-SDK/commit/165e833690b3dfeb437e3fd121d31551c025f615
